climada/hazard/test/test_trop_cyclone.py

```diff
@@ -26,25 +26,21 @@
 
 import numpy as np
 from scipy import sparse
-import xarray as xr
 
+from climada.hazard import test as hazard_test
```

Comment:
Was there a reason to change this? I find the old version clearer:
```suggestion
import climada.hazard.test as hazard_test
```

Reply:
No, this was unintentional.